src/Services/addToArchive.js

```diff
@@ -0,0 +1,20 @@
+import axios from "axios";
+
+const addToArchive = async (note, token, notesDispatch) => {
+	try {
+		const response = await axios.post(
+			`/api/notes/archives/${note._id}`,
+			{ note },
+			{ headers: { authorization: token } }
+		);
+		if (response.status === 201) {
+			notesDispatch({ type: "ARCHIVE_NOTE", payload: response.data });
+		} else {
+			throw new Error();
+		}
```

Comment:
You can also specify an error message to make it more understandable to user...What kind of error it is